test/e2e/network/util.go

```diff
@@ -58,13 +59,18 @@ func VerifySCTPModuleLoadedOnNodes(f *framework.Framework, nodes *v1.NodeList) b
 	cmd := "lsmod | grep sctp"
 	for _, node := range nodes.Items {
 		framework.Logf("Executing cmd %q on node %v", cmd, node.Name)
-		err := hostExec.IssueCommand(cmd, &node)
+		result, err := hostExec.IssueCommandWithResult(cmd, &node)
 		if err != nil {
 			framework.Logf("sctp module is not loaded or error occured while executing command %s on node: %v", cmd, err)
 			return false
 		} else {
-			framework.Logf("the sctp module is loaded on node: %v", node.Name)
-			return true
+			for _, line := range strings.Split(result, "\n") {
```

Comment:
do this block needs to be inside the `else`?
it seems that `if err != nil` it will return so no need to use it

Reply:
Fixed.